### DISCUSSION TITLE
Welcome to the Dracula Community

### DISCUSSION THREAD

**Discussion by zenorocha** [dracula/dracula-theme#400] [General]
There’s nothing more powerful than a community.

Communities allow us to connect, to share ideas, to help each other.

Communities are co-created - not by me, but by you.

![community](https://user-images.githubusercontent.com/398893/81713117-54c5ae00-942a-11ea-87d0-24c580012598.png)

Dracula started as one little theme, and it now has more than 100 themes. It’s not about an individual’s preferences anymore, it’s about all of us. That’s why this place exists.

In this community, we believe that coding should be fun. We believe that having a good theme keeps you focused. We believe that you should have a unified experience across all your apps.

Join us in this mission to create the most cross-platform theme ever created!

_Useful links_

* Read about the [origin of Dracula theme](https://draculatheme.com/about)
* Report bugs on the theme’s [GitHub Issues](https://github.com/dracula)
* Discover how to [create a new theme](https://draculatheme.com/contribute) using the official color palette

**Comment by avalonwilliams** [comment]
As a contributor to this awesome theme, I'm looking forward to engaging with the dracula community on this new platform! I'm sure that this will further our mutual cooperation and understanding as a community.

**Comment by oswaldoacauan** [comment]
![](https://media.giphy.com/media/12SXVd8bmXdSg0/giphy.gif)

**Comment by JimmyMultani** [comment]
Awesome! Looking forward to it!

**Comment by karlbateman** [comment]
When are you running another group buy for the GMK Dracula keycaps @zenorocha? I missed it 😭

**Reply by zenorocha** [reply]
I'm planning a surprise for Halloween xD
It's probably not going to include keyboards, but it's going to be nice.

**Reply by bigpick** [reply]
Oh man, big +1 on this. That [DM](https://dixiemech.com/gmkdracula) page just sits there and mocks me now 🤣

**Comment by pratayy** [comment]
while installing on ubuntu there are overlapping styles in the shell how can i fix that?

**Comment by sarangaswani** [comment]
What  does this exclamation mark mean?

![image](https://github.com/dracula/dracula-theme/assets/58489296/210d4130-6c65-4c7a-9d96-3b2f7cc400a0)

**Reply by luxonauta** [reply]
Please provide more context 😅

**Comment by josteink** [comment]
Hey. I just provided [a PR for a Dracula theme repo](https://github.com/dracula/visual-studio/pull/32) which don't seem to be very actively maintained.

What's the best way within the Dracula-community to ensure such things gets expedited? Are there like über maintainers one can CC, or does one just have to wait around and see if anything happens?

**Reply by luxonauta** [reply]
Thanks for the PR 💜

As the project is extensive, we depend on the community to maintain it. I've been keeping a few, so sometimes it takes a while to see everything. However, if you are interested, we are always open to new contributors and maintainers of themes for the apps.

**Comment by Socratesa** [comment]
Is it possible to make some improvements in C, I mean the macro and method names look the same color

**Comment by leandrofelix-dev** [comment]
Nice!

**Comment by Nemmen** [comment]
hey , i am a new user to this theme and i liked how its on the little dark side of the original dracula theme .

**Comment by nicolasRodosoft** [comment]
Hey guys just started using this theme is the a way to erase this blank space?
![Screenshot_1](https://github.com/dracula/dracula-theme/assets/162328023/c782283f-c10e-4e67-b37e-4310e9ae1e2e)
